app/main.py

```diff
@@ -2,5 +2,20 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
```

Comment:
You could use a decorator from `functools` for the `wrapper` to preserve the decorated function name and docstring